frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx

```diff
@@ -33,6 +34,14 @@ interface ModalProps {
 }
 
 function QueryActionCardContent({ action }: { action: WritebackQueryAction }) {
+  if (!action.dataset_query?.native?.query) {
+    return (
+      <CodeBlock>
+        <Icon name="warning" size={16} tooltip={t`No query found`} />
+      </CodeBlock>
+    );
+  }
+
```

Comment:
Ryan made some better error handling so that if the query is missing for whatever reason in the future it doesn't crash the page. In _theory_ this won't get hit